### PR TITLE
Create a multi user generator

### PR DESCRIPTION
**Pull request by MitchellGRead**
This is a different version of the workload generator that will create individual users each assigned to their workloads based off the workload file. The generator will then run each user concurrently with each user running their command sequentially to better simulate a real scenario of using our day trading system.

### CODE REVIEW

File: WorkloadGenerator/User.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        self.command_handler = UserCommands(ip, port, self.client)
+        self.commands = commands
+
+    async def processCommands(self):
+        resps = []
+        for command in self.commands:
+            print(f'{self.user_id} processing {command}')
+            # TODO Update so that this saves the amount of time taken to run each command and which command was run for analysis
+            resp = await self.command_handler.handleCommand(command)
+            resps.append(resp)
```
Comment (MitchPetersen) [inline WorkloadGenerator/User.py:19]: It'd probably be helpful to output the response received as well as the above print just so we can see if things are coming back as expected. At least for the debugging phase.

**Review by MitchPetersen — COMMENTED** [review]
Well done.

File: WorkloadGenerator/User.py (hunk 1 above)
Comment (MitchellGRead) [reply WorkloadGenerator/User.py:19]: I actually have this being covered in the UserCommand class instance the User creates where it prints the response each time :)

Comment (MitchPetersen) [reply WorkloadGenerator/User.py:19]: Don't know how I missed that. Yep.

**Pull request merged**